### PR TITLE
feat: Partner Show Opened notification page

### PR DESCRIPTION
**Pull request by olerichter00**
Addresses [ONYX-749]

## Description

This PR implements the new "Partner Show Opened" notification page.

## Screenshots

| web | mWeb |
| --- | --- |
|Screenshot 2024-04-08 at 10 13 57|Screenshot 2024-04-08 at 10 14 05 |

[ONYX-749]: https://artsyproduct.atlassian.net/browse/ONYX-749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Notifications/AlertNotification.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ export const AlertNotification: FC<AlertNotificationProps> = ({
 
       <Spacer y={1} />
 
-      <NotificationTypeLabel item={notificationData} />
+      <NotificationTypeLabel notification={notificationData} />
```
Comment (olerichter00) [inline src/Components/Notifications/AlertNotification.tsx:43]: Making `NotificationTypeLabel` a fragment container and changing the prop name.

File: src/Components/Notifications/Notification.tsx [hunk 2]
```diff
@@ -31,6 +32,7 @@ export const SUPPORTED_NOTIFICATION_TYPES = [
   "ARTICLE_FEATURED_ARTIST",
   "PARTNER_OFFER_CREATED",
   "VIEWING_ROOM_PUBLISHED",
+  "PARTNER_SHOW_OPENED",
```
Comment (olerichter00) [inline src/Components/Notifications/Notification.tsx:35]: We haven't released `PARTNER_SHOW_OPENED` yet in production, so I don't think we need an extra feature flag.

File: src/Components/Notifications/NotificationTypeLabel.tsx [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
-  item: {
-    notificationType: NotificationTypesEnum
-    publishedAt: string
-  }
+  notification: NotificationTypeLabel_notification$key
 }
 
-export const NotificationTypeLabel: React.FC<Props> = ({ item }) => {
-  const { notificationType } = item
+export const NotificationTypeLabel: React.FC<Props> = ({ notification }) => {
```
Comment (olerichter00) [inline src/Components/Notifications/NotificationTypeLabel.tsx:12]: Just refactoring this component to make it a fragment container.

File: src/Components/Notifications/NotificationPartnerShow.tsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
+import * as React from "react"
+import { graphql, useFragment } from "react-relay"
+import { RouterLink, RouterLinkProps } from "System/Router/RouterLink"
+import { AuthContextModule } from "@artsy/cohesion"
+import { Box, Button, Image, ResponsiveBox, Spacer, Text } from "@artsy/palette"
+import { NotificationPartnerShow_show$key } from "__generated__/NotificationPartnerShow_show.graphql"
+import { compact, truncate } from "lodash"
+import { CellShowStatus } from "Components/Cells/CellShow"
+
+const MAX_WIDTH = 600
```
Comment (nickskalkin) [inline src/Components/Notifications/NotificationPartnerShow.tsx:10]: **suggestion / non-blocking**: there is a constant for max width [here](https://github.com/artsy/force/blob/main/src/Components/Notifications/constants.ts), let's use it?

File: src/Components/Notifications/NotificationPartnerShow.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+  show: showProp,
+  contextModule,
+  onClick,
+  ...rest
+}) => {
+  const show = useFragment(notificationShowFragment, showProp)
+
+  const image = show?.coverImage?.cropped
+
+  if (!image) {
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply src/Components/Notifications/NotificationPartnerShow.tsx]: Already removed, just forgot to commit 👍

**Review by nickskalkin — APPROVED** [review]
great work 👍

**Pull request merged**